src/main/java/DAO/UserDAO.java

```diff
@@ -0,0 +1,46 @@
+package DAO;
+
+import Entity.User;
+import org.hibernate.Session;
+import persistence.HibernateSessionFactoryUtil;
+import java.util.List;
+
+public class UserDAO {
+
+    private static UserDAO userDAO;
+
+    public static UserDAO instance() {
+        if (userDAO == null) {
+            userDAO = new UserDAO();
+        }
+        return userDAO;
+    }
+
+    private UserDAO() {
+    }
+
+    public void addUserToDB(String login, String password) {
+        if ((login == null) || (password == null) || (login.equals("")) || (password.equals(""))) {
+            return;
+        }
+        Session session = HibernateSessionFactoryUtil.getSessionFactory().openSession();
```

Comment:
we can move in main class:
HibernateSessionFactoryUtil.getSessionFactory()

then we can create constructor with parameter UserDao(...) and pass this line here;